charts/metallb/values.yaml

```diff
@@ -177,13 +177,13 @@ prometheus:
             severity: warning
```

Comment:
Issue #1019 suggests also `Avoid creating the same alert with the same severity but different thresholds`
So should we also add modification to the `thresholds` of `addressPoolUsage`?
i.e maybe just having one threshold for warning at 80 percent instead of two warning thresholds?

Reply:
besides that, the other changes look good! :)

Reply:
@tylerauerbeck WDYT about the thresholds? can you combine them, please?

Reply:
I'd still keep the lowest as 75, just to be consistent across releases. I am not sure what the severity levels are but if possible I'd keep the 3 ranges. If not, let's zap the one in the middle